### ISSUE TITLE
Deprecation warning on macOS 12.0: kUTTypeURL

### ISSUE THREAD

**Issue by dsieger** [glfw/glfw#2003]
Hi,

Compiling glfw (3.3.5 and master) on macOS 12.0.1 with Xcode 13.0.0 results in the following deprecation warning:

```
glfw/src/cocoa_window.m:366:62: warning: 'kUTTypeURL' is deprecated: first deprecated in macOS 12.0 - Use UTTypeURL instead. [-Wdeprecated-declarations]
        [self registerForDraggedTypes:@[(__bridge NSString*) kUTTypeURL]];
                                                             ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.0.sdk/System/Library/Frameworks/CoreServices.framework/Frameworks/LaunchServices.framework/Headers/UTCoreTypes.h:210:26: note: 'kUTTypeURL' has been explicitly marked deprecated here
extern const CFStringRef kUTTypeURL                                  API_DEPRECATED("Use UTTypeURL instead.", ios(3.0, 15.0), macos(10.4, 12.0), tvos(9.0, 15.0), watchos(1.0, 8.0));
                         ^
```

Could be fixed by using `UTTypeURL` as indicated plus including and linking the UniformTypeIdentifiers framework. But then, this is not really my area.

Thanks,
Daniel

**Comment by endaye** [comment]
same issue.

**Comment by elmindreda** [comment]
This should be fixed now with 7f6aa587f8a0640b055133eddf09c9a652c94234. Thank you for reporting this!
